offer-manager-domain/src/main/groovy/life/qbic/business/products/archive/ArchiveProduct.groovy

```diff
@@ -0,0 +1,12 @@
+package life.qbic.business.products.archive
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since: <versiontag>
```

Comment:
```suggestion
 * <h1>4.3.1 Archive Service Product</h1>
 * <br>
 * <p> The use case archives unused or old service products. These products cannot be added to new offers but still show in old offers.</p>
 * <br>
 *
 * @since: 1.0.0
```
Did I get that right?